pipelines/qp_images.yml

```diff
@@ -0,0 +1,37 @@
+resources:
+  - name: qp-images
+    type: git
+    source:
+      branch: master
+      uri: git@github.com:Pivotal-DataFabric/qp-images.git
+      private_key: {{qp_images_deploy_key}}
```

Comment:
I don't think we need a deploy key. This is a public repo. But we can ship it before changing this bit.